code/game/gamemodes/blob/overmind.dm

```diff
@@ -91,10 +91,13 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 
 		if(!Ablob.blob_allowed)
 			continue
-
-		playsound(L, 'sound/effects/splat.ogg', 50, 1)
-		L.death()
-		new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		
+		if(!("blob" in L.faction) || !istype(L, /mob/living/simple_animal/hostile/blob))
```

Comment:
Do we not have a proc for this?

Reply:
for the blob check or the faction check

Reply:
Faction

Reply:
https://github.com/tgstation/tgstation/blob/626302c1bf4bb2a37aa6508145751f6c4c78696d/code/modules/mob/mob.dm#L827

there's faction_check on mob but I don't think it's relevant

Reply:
You shouldn't need the istype after the faction check